CONTRIBUTING.md

```diff
@@ -103,3 +102,58 @@ developers.
 
   More information about tool development can be found [on the
   wiki](https://wiki.galaxyproject.org/Develop).
+
+## Handling Pull Requests
+
+Everyone is encouraged to express opinions and issue non-binding votes on pull
+requests, but only members of the *committers* group may issue binding votes
+on pull requests. Information on the *committers* group can be found in the
+[organization document](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst)
+describing governance of the core Galaxy code base.
+
+Votes on pull requests should take the form of
+[+1, 0, -1, and fractions](http://www.apache.org/foundation/voting.html)
+as outlined by the Apache Foundation.
+
+Pull requests modifying pre-existing releases should be restricted to bug fixes
+and require at least 2 *+1* binding votes from someone other than the author of
+the pull request with no *-1* binding votes.
+
+Any pull request changing or clarifying the procedures governing this
+repository must remain open for at least 4 days and require at least 3 *+1*
```

Comment:
@dannon Like I mentioned - I don't think we got 10 +1 for any vote yet when it was a simple as raising your hand and gathering votes for these things will be much harder on github.

If I understood the ways in which this was inadequate or how situations will change - I think we would just modify it now. The fact is these procedures may be wholly terrible - we really don't know. Finally, I would wait for the current rule change procedures to actually fail us in someway before limiting our ability to modify them.

How about 4 +1s and the 192 hours?